.travis.yml

```diff
@@ -47,9 +47,10 @@ jobs:
 
       install:
         - pip install -r docs/requirements.txt
+        - python setup.py install
 
       script:
-        - make -C docs/
+        - make clean -C docs/ html
```

Comment:
Was this the error?

Reply:
Yeah

Reply:
Hmm weird, never done this before

Reply:
sphinx-quickstart creates a makefile to automatically build the rst files